.ci/scripts/ecr-images.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+set -euxo pipefail
+IMAGE_SUFFIX=$1
+ECR_REGISTRY=$2
+ELASTIC_AGENT_DOCKER_TAG=$(make get-ci-agent-version)
+
+
+
+load_and_push_image () {
+    docker load -i /tmp/"$1"-"${IMAGE_SUFFIX}".tar
+    if [ $1 != "elastic-agent" ]; then
+        docker tag "$2" $ECR_REGISTRY/"$2"
+    fi
+    docker push $ECR_REGISTRY/"$2"
+}
+
+load_and_push_image cloudbeat "cloudbeat:latest" &
+load_and_push_image pytest "cloudbeat-test:latest" &
+load_and_push_image elastic-agent "elastic-agent:$ELASTIC_AGENT_DOCKER_TAG"
```

Comment:
when using `latest` tag you will override the same image whenever the script is running
as a way to keep things organized, it is better to tag with the commit_sha
this way you are making sure you the tests use the expected image and not accidentally using  an unexpected version of it

Reply:
see `GITHUB_SHA` https://docs.github.com/en/actions/learn-github-actions/environment-variables

Reply:
I agree that we need to replace `latest` tag by `commit_sha`. Since we are using different scripts and tools in CI flow, for replacing `latest` tag we need to open a separate task. Currently the `latest` tag is used in:
- `make`
- `just`
- `kustomize`
- `helm`
- test infra

Reply:
New [task](https://github.com/elastic/security-team/issues/5694) is opened